vue/components/ui/organisms/listing/listing.vue

```diff
@@ -282,8 +283,11 @@ export const Listing = {
         getFilter() {
             return this.$refs.filter;
         },
-        onLineupClick(item, index) {
-            this.$emit("click:lineup", item, index);
+        onLineupRowClick(item, index) {
+            this.$emit("click:lineup:row", item, index);
+        },
+        onFilterTableRowClick(item, index) {
+            this.$emit("click:filter:table:row", item, index);

```

Comment:
`click:table:row`, it's not relevant for the client of this that the click came from the filter since in fact that's just a wrapper around the table to add functionality.